packages/espressocash_app/lib/features/activities/widgets/activity_tile.dart

```diff
@@ -30,10 +31,12 @@ class CpActivityTile extends StatelessWidget {
     final incomingAmount = this.incomingAmount;
     final outgoingAmount = this.outgoingAmount;
 
+    final iconSize = Provider.of<double?>(context, listen: false) ?? 42.0;
```

Comment:
Don't do this, it's not maintainable. Also, in design I don't see icons in recent activity block at all.

Reply:
Updated to hide icon via widget constructor bool